app/src/main/java/framgia/com/ichat/data/model/User.java

```diff
@@ -0,0 +1,19 @@
+package framgia.com.ichat.data.model;
+
+public class User {
+    private String mEmail;
+    private String mPassword;
+
+    public User(String mEmail, String mPassword) {
```

Comment:
tip số 6
https://viblo.asia/p/mot-so-tips-tricks-khi-lam-viec-voi-android-studio-yMnKMJjgZ7P
em làm theo hướng dẫn rồi xoá getter setter constructor đi gen lại 
```suggestion
    public User(String email, String password) {
```